pages/src/source/tutorials/schematics.md

```diff
@@ -393,3 +404,79 @@ Put the [deco controller](../../source/items/decocontroller) somewhere in the sc
 {% endversion %}
 
 ![Upgradable Decos](../../assets/images/tutorial/upgradabledecos.png)
+
+
+### How to use custom mineshafts in style packs?
+
+The size must be 9 x 4 x 9 blocks. Rotation must match the Mine in the Style Pack. Use an existing style pack as a template along with the [scan tool](../../source/items/scantool) to create the blueprints.
+
+{% version "1.18.2" before=true %}
+The custom mineshafts need to be at `schematics/yourstyle/miner/*`.
+{% endversion %}
+
+{% version "1.19" after=true %}
+The custom mineshafts need to be at `blueprints/yourstyle/infrastructure/mineshafts/*.
+
+It's recommended that you use the [tag tool and Tag Anchor](../../source/items/tagtool) to make the mineshafts `invisible`.  Take care that the anchor is in the same position as in the original mineshafts -- the very center bottom block.
+{% endversion %}
+
+{% version "1.19" after=true %}
+### How to make custom quarries in style packs?
```

Comment:
Is there a reason to make this section 1.19+ only? Just the naming part is 1.19+ afaik (1.18 had them directly in the style folder iirc)

Reply:
Mostly just because I couldn't remember how they worked in 1.18 so I didn't want there to be incorrect info.